src/components/Sidebar/SidebarProjectIcon.js

```diff
@@ -14,6 +14,7 @@ type Props = {
   color?: string,
   iconSrc?: string,
   isSelected: boolean,
+  handleSelect: () => any,
```

Comment:
Could this be `() => void` if you don’t use the return value?

Reply:
yeah, good point. Will update.